seed/analysis_pipelines/better.py

```diff
@@ -0,0 +1,448 @@
+                'version': '2.2.0'
+            },
+            E.Facilities(
+                E.Facility(
+                    {'ID': 'Facility-1'},
+                    E.Sites(
+                        E.Site(
+                            {'ID': 'Site-1'},
+                            E.Buildings(
+                                E.Building(
+                                    {'ID': 'Building-1'},
+                                    E.PremisesName(property_state.property_name),
+                                    E.OccupancyClassification("Office"),
+                                    E.eGRIDRegionCode(eGRIDRegion),
+                                    E.Address(
+                                        E.City(property_state.city),
+                                        E.State(property_state.state),
+                                        E.PostalCode(property_state.postal_code)
+                                    ),
+                                    E.FloorAreas(
+                                        E.FloorArea(
+                                            E.FloorAreaType("Gross"),
+                                            E.FloorAreaValue(gross_floor_area)
+                                        )
+                                    ),
+                                    E.PremisesIdentifiers(
+                                        E.PremisesIdentifier(
+                                            E.IdentifierLabel('Custom'),
+                                            E.IdentifierCustomName(PREMISES_ID_NAME),
+                                            E.IdentifierValue(str(analysis_property_view.id)),
+                                        )
+                                    ),
+                                    E.Longitude(str(analysis_property_view.property_state.longitude)),
+                                    E.Latitude(str(analysis_property_view.property_state.latitude)),
+                                )
+                            )
+                        )
+                    ),
+                    E.Reports(
+                        E.Report(
+                            {'ID': 'Report-1'},
+                            E.Scenarios(
+                                E.Scenario(
+                                    {'ID': 'Scenario-Measured'},
+                                    E.ScenarioType(
+                                        E.CurrentBuilding(
+                                            E.CalculationMethod(
+                                                E.Measured()
+                                            )
+                                        )
+                                    ),
+                                    E.ResourceUses(
+                                        E.ResourceUse(
+                                            {'ID': 'Resource-' + str(11)},
```

Comment:
I'm guessing these IDs will be dehardcoded later?

Reply:
That is the intent.  Ideally, this ID would be populated by the meter type dynamically but I had trouble finding a solution that would filter back up to `ResourceUses` outside the context of the nested `for`.  Definitely open to suggestions or a quick pair to talk about strategy.